PlutoHelperAgent/AppDelegate.m

```diff
@@ -98,8 +98,8 @@ -    (void)getUrl:(NSAppleEventDescriptor *)event
         
         NSString *projectid = [parts objectAtIndex:2];
         
-        [ProjectLockerAndKeychainFunctions get_data_from_server:@"%@/api/project/" :@"/files" :projectid completionHandler:^void (NSURLResponse *response,NSDictionary *filesResult){
-            [ProjectLockerAndKeychainFunctions get_data_from_server:@"%@/api/storage/" :NULL :filesResult[@"files"][0][@"storage"] completionHandler:^(NSURLResponse *response, NSDictionary *storageResult) {
+        [ProjectLockerAndKeychainFunctions get_data_from_server:@"/api/project/" :@"/files" :projectid completionHandler:^void (NSURLResponse *response,NSDictionary *filesResult){
+            [ProjectLockerAndKeychainFunctions get_data_from_server:@"/api/storage/" :NULL :filesResult[@"files"][0][@"storage"] completionHandler:^(NSURLResponse *response, NSDictionary *storageResult) {
```

Comment:
is this safe? Would a NULL value not cause `ProjectLockerAndKeychainFunctions.m` to crash at line 111?

Reply:
No. This is not a problem.

Reply:
ok. what happens in this eventuality?

Reply:
The string is correctly constructed.

The line you mentioned is irrelevant to the to the arguments of get_data_from_server. To see what happens, have a look at line 218.